docs/source/getting_started/active_learning.rst

```diff
@@ -101,23 +101,23 @@ Then, in your Python script, you will need to initialize the `ApiWorkflowClient`
    and tell the `ActiveLearningAgent` to only sample from this tag. To do so, set
    the `query_tag_name` argument in the constructor of the agent.
 
-Let's configure the sampling request and request an initial selection next:
+Let's configure the selection request and request an initial selection next:
```

Comment:
there's a repetition of words here now so we should probably rewrite this sentence.